blockchain/blockchain.go

```diff
@@ -558,16 +597,18 @@ func stateUpdateByHash(txn db.Transaction, hash *felt.Felt) (*core.StateUpdate,
 }
 
 // SanityCheckNewHeight checks integrity of a block and resulting state update
-func (b *Blockchain) SanityCheckNewHeight(block *core.Block, stateUpdate *core.StateUpdate, newClasses map[felt.Felt]core.Class) error {
+func (b *Blockchain) SanityCheckNewHeight(block *core.Block, stateUpdate *core.StateUpdate,
+	newClasses map[felt.Felt]core.Class,
```

Comment:
Is there a reason why we aren't adding the commitments to the `core.Header` struct? ~We can calculate them in `feeder2core.AdaptBlock`.~